config/verificationMail.js

```diff
@@ -43,6 +48,85 @@ const config = {
         message: 'Failed to send the email'
       };
     }
+  },
+
+  async sentResetMail(params) {
+    const client = nodemailer.createTransport({
+      service: process.env.TRANSPORT_SERVICE,
+      auth: {
+        user: process.env.TRANSPORT_USER,
+        pass: process.env.TRANSPORT_PASS
+      }
+    });
+
+    const source = fs.readFileSync(path.join(__dirname, '../template/reset.hjs'), 'utf8');
+    const template = Handlebars.compile(source);
+    const { username, email } = params;
+    const tokenToSend = await jwt.sign({ email }, process.env.SECRET);
+    const url = `${process.env.SERVER_ADDRESS}/api/v1/update_password/${tokenToSend}`;
+    const envelope = {
+      from: process.env.SENDER,
+      to: email,
+      subject: 'Reset password',
+      html: template({ username, url })
+    };
+    try {
+      return await client.sendMail(envelope);
+    } catch (error) {
+      return {
+        status: 500,
+        message: 'Failed to send the email'
+      };
+    }
+  },
+
+  async sentNotificationMail(params) {
+    const client = nodemailer.createTransport({
+      service: process.env.TRANSPORT_SERVICE,
+      auth: {
+        user: process.env.TRANSPORT_USER,
+        pass: process.env.TRANSPORT_PASS
+      }
+    });
+
+    const source = fs.readFileSync(path.join(__dirname, '../template/notification.hjs'), 'utf8');
+    const template = Handlebars.compile(source);
+    const {
+      subscribeTo, username, slug, action
+    } = params;
+    const url = `${process.env.SERVER_ADDRESS}/api/v1/activation/${slug}`;
+
+    const getSubscriber = await subscribers.findOne({
+      where: {
+        [Op.or]: [{ articleId: subscribeTo }, { authorId: subscribeTo }]
+      },
+      attributes: { subscribers }
+    });
+
+    try {
+      getSubscriber.subscribers.forEach(async (subId) => {
+        const subs = await User.findOne({
+          where: { id: subId }
+        });
+
+        const receiver = subs.username;
+
+        const envelope = {
+          from: process.env.SENDER,
+          to: subs.email,
+          subject: 'new article',
```

Comment:
The purpose of this message I don't think it's for notifying a user that new article created. It's better if you find a new subject message.